go/tools/sizegen/integration/integration_test.go

```diff
@@ -60,17 +59,17 @@ func TestTypeSizes(t *testing.T) {
 		{&Slice3{field1: []*Bimpl{{}, {}, {}, {}}}, SliceHeaderSize + PtrSize*4 + 8*4},
 
 		{&Map1{field1: nil}, PtrSize},
-		{&Map1{field1: map[uint8]uint8{}}, PtrSize + HashMapHeaderSize},
-		{&Map1{field1: map[uint8]uint8{0: 0}}, PtrSize + HashMapHeaderSize + BucketHeaderSize + 1*BucketSize + 1*BucketSize + PtrSize},
+		{&Map1{field1: map[uint8]uint8{}}, 56},
+		{&Map1{field1: map[uint8]uint8{0: 0}}, 80},
 
 		{&Map2{field1: nil}, PtrSize},
-		{&Map2{field1: map[uint64]A{}}, PtrSize + HashMapHeaderSize},
-		{&Map2{field1: map[uint64]A{0: {}}}, PtrSize + HashMapHeaderSize + BucketHeaderSize + 8*BucketSize + 16*BucketSize + PtrSize},
+		{&Map2{field1: map[uint64]A{}}, 56},
+		{&Map2{field1: map[uint64]A{0: {}}}, 264},
 
 		{&Map3{field1: nil}, PtrSize},
-		{&Map3{field1: map[uint64]B{}}, PtrSize + HashMapHeaderSize},
-		{&Map3{field1: map[uint64]B{0: &Bimpl{}}}, PtrSize + HashMapHeaderSize + BucketHeaderSize + 8*BucketSize + FatPointerSize*BucketSize + PtrSize + 8},
-		{&Map3{field1: map[uint64]B{0: nil}}, PtrSize + HashMapHeaderSize + BucketHeaderSize + 8*BucketSize + FatPointerSize*BucketSize + PtrSize},
+		{&Map3{field1: map[uint64]B{}}, 56},
+		{&Map3{field1: map[uint64]B{0: &Bimpl{}}}, 272},
+		{&Map3{field1: map[uint64]B{0: nil}}, 264},
```

Comment:
@vmg Made these explicit for now, do you have a better idea for these?

Reply:
Nah this makes perfect sense. It'll help us catch errors if they were to change the internal sizing of the data structures between Go releases.